src/crab/interval.cpp

```
@@ -17,6 +17,11 @@ static Interval make_dividend_when_both_nonzero(const Interval& dividend, const
     return dividend + Interval{1} - divisor;
 }
 
+static ExtendedNumber divide_bound_by_negative_singleton(const ExtendedNumber& bound, const Number& divisor) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)